src/pydot/dot_parser.py

```diff
@@ -383,6 +383,29 @@ def push_node_stmt(s: str, loc: int, toks: ParseResults) -> "pydot.core.Node":
     return n
 
 
+class HTML(Token):
+    def __init__(self) -> None:
+        super().__init__()  # type: ignore
+
+    def parseImpl(
+        self, instring: str, loc: int, do_actions: bool = True
+    ) -> T.Tuple[int, str]:
+        start = loc
+        if not (loc < len(instring) and instring[loc] == "<"):
+            raise ParseException(instring, loc, "expected <", self)
+        open = 1
+        loc += 1
+        while loc < len(instring):
+            if instring[loc] == "<":
+                open += 1
+            elif instring[loc] == ">":
+                open -= 1
+            loc += 1
+            if open == 0:
+                return loc, instring[start:loc]
+        raise ParseException(instring, loc, "expected >", self)
```

Comment:
The uncovered line here is fine; we probably _should_ have more failure-tests, but at the moment we don't have that many.

And I wouldn't even want to go too hard into populating them as unittest `assertRaises` calls, until we finally get all of the testing rewritten in pytest.